content/doc/book/pipeline/index.adoc

```diff
@@ -6,43 +6,138 @@ layout: chapter
 :author:
 :email: jenkinsci-users@googlegroups.com
 :sectanchors:
-:toc: left
+:toc:
 
 = Pipeline
 
-This chapter will cover all aspects of the Jenkins Pipeline, from running pipeline jobs
-to writing your own pipeline code.
+This chapter will cover all aspects of Jenkins Pipeline, from running pipeline jobs
+to writing your own pipeline code, and even extending Pipeline.
 
 This chapter is intended to be used by Jenkins users of all skill levels,
 but beginners may need to refer to some sections of "<<using#,Using Jenkins>>"
 to understand some topics covered in this chapter.
 
-The sections are ordered from simpler to progressively more complex subjects.
-The earlier sections are intended for those new to pipeline or unfamiliar with its
-latest features.  The later sections with discuss expert-level considerations
-and corner-cases.
-
 If you are not yet familiar with basic Jenkins terminology and features, start with
 <<getting-started#,Getting Started with Jenkins>>.
 
-If you are already familiar with Jenkins basics and would like to delve deeper
-into generally how to use various features, see
-<<using#,Using Jenkins>>.
+[[overview]]
+== What is Pipeline?
+
+Jenkins Pipeline is a suite of plugins which supports implementing and
+integrating continuous delivery pipelines into Jenkins Pipeline provides an
+extensible set of tools for modeling simple-to-complex delivery pipelines "as
+code" via the Pipeline DSL.
+footnoteref:[dsl,link:https://en.wikipedia.org/wiki/Domain-specific_language[Domain-Specific Language]]
```

Comment:
Not sure either.  Just trying to keep the intent, while getting rid of the bit about "simple-to-complex delivery pipelines".